### PR TITLE
Add Scheme.uses_table

### LINKED ISSUE #280: Property to check if scheme uses a misc table
Currently, we use `isinstance(scheme.labels, str)` to check if a scheme is linked to a misc table. To make the code easier to read we should think about a property, maybe something like  `Scheme.is_table`?

### PR DESCRIPTION
**Pull request by frankenjoe**
Closes #280 

image

### CODE REVIEW

**Comment by frankenjoe** [comment]
Turned out there are not so many places within `audformat` where we can actually use the property, but I think it will also make code outside of the package easier to understand, e.g. in https://github.com/audeering/audb/pull/222

**Comment by hagenw** [comment]
Yes, I also think it will be helpful outside of `audformat` even if not used to much inside `audformat` itself.

What I'm not completely convinced yet is the name. `has_table` somehow implies that the scheme has a table, which is not completely correct as the labels of the scheme are stored in a misc table that exists outside of the scheme.

One alternative would be to start the name also with `labels` to make it appear near the `labels` attribute in the documentation, e.g. `labels_stored_in_table`.

Otherwise, maybe `uses_table` might be better than `has_table`?

**Comment by frankenjoe** [comment]
Ok, I would prefer `uses_table`  over `labels_stored_in_table`.

**Comment by hagenw** [comment]
> Ok, I would prefer `uses_table` over `labels_stored_in_table`.

Good, I'm fine with `uses_table`

**Comment by frankenjoe** [comment]
I renamed the property to `uses_table`

File: audformat/core/scheme.py [hunk 1]
```diff
@@ -164,12 +164,29 @@ def __init__(
         self.maximum = maximum if self.is_numeric else None
         r"""Maximum value"""
 
+    @property
+    def uses_table(self) -> bool:
+        r"""Scheme has labels stored in a table.
```
Comment (hagenw) [inline audformat/core/scheme.py:169]:
```suggestion
        r"""Scheme has labels stored in a misc table.
```

File: audformat/core/scheme.py [hunk 2]
```diff
@@ -164,12 +164,29 @@ def __init__(
...
+
+        If property is ``True``
+        the attribute ``labels``
+        is set to an ID of a
+        :class:`audformat.MiscTable`
+        where the actual label
+        values are stored.
+
+        Returns:
+            ``True`` if scheme has labels stored in a table
```
Comment (hagenw) [inline audformat/core/scheme.py:179]:
```suggestion
            ``True`` if scheme has labels stored in a misc table
```

**Pull request merged**